src/rpc/rawtransaction.cpp

```diff
@@ -947,12 +947,20 @@ static RPCHelpMan testmempoolaccept()
 
     TxValidationState state;
     bool test_accept_res;
-    CAmount fee;
+    CAmount fee{0}; // To return transaction fee from AcceptToMemoryPool
     {
         LOCK(cs_main);
         test_accept_res = AcceptToMemoryPool(mempool, state, std::move(tx),
             nullptr /* plTxnReplaced */, false /* bypass_limits */, max_raw_tx_fee, /* test_accept */ true, &fee);
     }
+
+    // TODO: move ATMP absurdFee check to here
```

Comment:
in commit f907f4b9389b86fc655f45917367be5838b79684:

Any reason to not squash commit 6a4a7c64246c826fed6b2bdf97ad91a5f864621b into this? Seems odd to add an TODO in one commit and then fix it in the next commit in the same pull.

Reply:
Yeah, I had this odd idea that people might review commit-by-commit and that documenting what to expect might be helpful, but perhaps it's just weird and confusing 😅

Reply:
will squash, that is a better way 👍